basyx.common/basyx.filerepository-backend/src/test/java/org/eclipse/digitaltwin/basyx/core/filerepository/backend/FileRepositoryTestSuite.java

```diff
@@ -124,23 +101,26 @@ public void deleteFile() {
 	@Test(expected = FileDoesNotExistException.class)
 	public void deleteNonExistingFile() {
 		FileRepository fileRepo = getFileRepository();
+		
 		String filePath = "does not exist";
 		
 		fileRepo.delete(filePath);
 	}
 	
 	@Test
-	public void doesExists() {
+	public void fileDoesExists() {
 		FileRepository fileRepo = getFileRepository();
-		String filePath = getSavedTestFile(fileRepo);
+		
+		FileMetadata testFileMetadata = createDummyFile();
```

Comment:
I think there is no need to use testSuffix, a simple fileMetadata is sufficient.

Please change it everywhere.

Reply:
changed it